README.md

```diff
@@ -139,6 +144,13 @@ Data viewers/Editors based on shapes.
 - SHACL-related specifications
   - [The Shape Topologies algorithm](https://treecg.github.io/specification/shape-topologies)
   - [DASH Data Shapes](https://www.datashapes.org/) - Platform-independent extensions of SHACL for common tasks. Stuff that could become an official standard in the future.
+ 
+- SHACL on Graphs (as discussed in [shacl#22](https://github.com/w3c/shacl/issues/22) and [Inst4CIM-KG#140](https://github.com/Sveino/Inst4CIM-KG/issues/140))
+  - [Data Graph](https://w3c.github.io/data-shapes/shacl/#data-graph) and Shapes Graph (SHACL spec) - the inputs to a SHACL validator, but these are "ephemeral", and spec doesn't concretize how to pass them to the validator.
```

Comment:
```suggestion
  <!--lint ignore double-link-->
  - [Data Graph](https://w3c.github.io/data-shapes/shacl/#data-graph) - The input to a SHACL validator (along with the Shapes Graph or SHACL spec), but these are "ephemeral", and spec doesn't concretize how to pass them to the validator.
```

Reply:
hi @amivanoff , so it ignores the fragment when checking for duplicated links?

Reply:
Yes, it does. We can enable/disable separate linter's rulesets per list item (or globally).

Just put it after "lint ignore" and separate with a space from each other.

- double-link -- checks list item duplication
- awesome-list-item -- checks list item formatting

Examples:

```
<!--lint ignore double-link-->
<!--lint ignore awesome-list-item-->
<!--lint ignore awesome-list-item double-link-->
```

Reply:
The awesome linter "ignores the fragment when checking for duplicated links"? That's not awesome! It should result in feedback to the linter project! Different fragments are different URIs, even if dereferencing both results in the browser displaying approximately the same information.